src/photos/components/AppRoute.jsx

```diff
@@ -5,21 +5,21 @@ import Layout from './Layout'
 import Timeline from '../containers/Timeline'
 import AlbumsView from '../containers/AlbumsView'
 import AlbumPhotos from '../containers/AlbumPhotos'
-import Viewer from '../components/Viewer'
+import PhotosViewer from '../components/PhotosViewer'
 import { PhotosPicker } from '../ducks/albums'
 
 export const ComingSoon = () => <p style="margin-left: 2em">Coming soon!</p>
 
 const AppRoute = (
   <Route component={Layout}>
     <Route path="photos" component={Timeline}>
-      <Route path=":photoId" component={Viewer} />
+      <Route path=":photoId" component={PhotosViewer} />
```

Comment:
Can't we have the same `FileViewer` for `photos` app?
Or the `PhotosViewer` for the `drive` app?

Reply:
They have a slightly different behavior, that could diverge even more. That's why I preferred having separate wrappers

Reply:
Look good to me, but can we rename with the app name with one of these:
- s/`FileViewer`/`DriveViewer/` and `PhotosViewer`
- s/`FileViewer`/`FilesViewer/` and `PhotosViewer`
- `FileViewer` and s/`PhotosViewer`/`PhotoViewer/`

Reply:
ok for `FilesViewer` and `PhotosViewer`, but what about make their names even more explicit like `FilesViewerWrapper` or something?